Eatery/Controllers/EateryTabBarController.swift

```diff
@@ -60,17 +60,18 @@ class EateryTabBarController: UITabBarController {
     ) -> UIInterfaceOrientation {
         .portrait
     }
+    
+    func preselectEatery(withName name: String) {
```

Comment:
Where is this being called from?

Reply:
This is being called from called handleShortcutItem() in lines 77-81 of the AppDelegate

Reply:
Isn't the AppDelegate calling `eateryTabBarController.eateriesViewController.preselectEatery(withName: shortcutItem.type)`? So it's bypassing this method?